src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -246,103 +233,6 @@ export class Flyout extends Component<FlyoutProps, FlyoutState> {
     }
   };
 
-  legacyImport = async () => {
-    const { serviceRegistry, indexPatterns, overlays, http, allowedTypes } = this.props;
-    const { file, importMode } = this.state;
-
-    this.setState({ status: 'loading', error: undefined });
-
-    // Log warning on server, don't wait for response
-    logLegacyImport(http);
```

Comment:
The method and route will be removed in a follow up that handles the second bullet point of https://github.com/elastic/kibana/issues/103921